### PR TITLE
Warn users when they try to ping here or everyone

### PR DESCRIPTION
**Pull request by dylangarcia**
image

### CODE REVIEW

File: src/features/commands.ts [hunk 1]
```diff
@@ -410,6 +412,30 @@ To integrate it into your editor: https://prettier.io/docs/en/editors.html`,
...
+
+      if (!member || isStaff(member)) {
+        return;
+      }
+
+      msg.channel.send({
+        embed: {
+          title: "Tsk tsk.",
+          type: "rich",
+          description: `Please do **not** try to use \`@here\` or \`@everyone\` - there are ${msg.guild.memberCount} members in Reactiflux. Everybody here is a volunteer, and somebody will help you if they can.`,
```
Comment (bpas247) [inline src/features/commands.ts:434]: Although some people also try to ping everyone when they've made something and they're trying to share it in `#i-made-this`, maybe we could also add a short blurb for that scenario?

Comment (dylangarcia) [reply src/features/commands.ts:434]: `Somebody will respond if they can`?

Comment (bpas247) [reply src/features/commands.ts:434]: works for me

**Review by bpas247 — APPROVED** [review]
🚢

**Pull request merged**